packages/application-shell/src/components/custom-view-shell/custom-view-shell.tsx

```diff
@@ -205,8 +219,14 @@ function CustomViewShell(props: TCustomViewShellProps) {
       : hostContext.hostUrl;
 
   return (
-    <div data-extension-type={CUSTOM_EXTENSION_TYPES.CUSTOM_VIEW}>
+    <div
+      css={css`
+        height: 100%;
+      `}
+      data-extension-type={CUSTOM_EXTENSION_TYPES.CUSTOM_VIEW}
+    >
```

Comment:
I guess this can also be removed now...?

Reply:
Totally.

Removed here: 1b5f4702b075c058d749fef1287db758ab9d6088